Assets/MixedRealityToolkit-Examples/Graph/GraphConnectorTestAsync.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Graph;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Examples
+{
+    /// <summary>
+    /// Class to test GraphConnector using async model.
+    /// </summary>
+    public class GraphConnectorTestAsync : MonoBehaviour
+    {
+        /// <summary>
+        /// Called by Unity on the frame when the script is enabled just before any of the Update methods is called the first time.
+        /// </summary>
+        private async void Start()
```

Comment:
Ah there's two! disregard my last comment then 😄

Reply:
Since we're specifying that vNEXT requires .NET 4.x support we can take out the IEnumerator version of this class.